src/cs50.h

```diff
@@ -140,8 +168,9 @@ long long GetLongLong(void) __attribute__((deprecated));
  * upon error or no input whatsoever (i.e., just EOF). Stores string
  * on heap, but library's destructor frees memory on program's exit.
  */
-string get_string(struct prompt *p);
+
+string get_string(bool _internal, char const *fmt, ...) __attribute__((format(printf, 2, 3)));
```

Comment:
Would it be more idiomatic to have `v*` functions, isolated to `cs50.c`, that receive the `va_list` from `get_*` (whose signatures could thus remain unchanged)? Cf. http://c-faq.com/varargs/handoff.html.